ldap/servers/slapd/ldapi.c

```diff
@@ -201,6 +201,7 @@ slapd_bind_local_user(Connection *conn)
                                                         NULL, NULL, NULL, NULL);
                         }
```

Comment:
I think we might have another leak here for the case when account is locked, auth_dn is never passed to `bind_credentials_set_nolock()` (which assigns it to `conn`), nor is it explicitly freed.

Reply:
`root_dn` is also leaked when root account is locked. I've added tests to verify these leaks under ASAN.